api-service/crowdsource-webapp/src/app/relative-toxicity-job/relative-toxicity-job.component.html

```diff
@@ -22,24 +15,33 @@ <h1>Comparative Toxicity Job</h1>
   </div>
 
   <div *ngIf="question">
-    <div class="id" *ngIf="notEmbedded">Question Id: <span>{{question.id}}</span></div>
-
-    <div class="experimentToggle">
-      <mat-slide-toggle labelPosition="before "[(ngModel)]="useRadioButtons">Use radio buttons</mat-slide-toggle>
+    <div class="experimentToggle" *ngIf="debugMode">
+      <mat-slide-toggle labelPosition="before" [(ngModel)]="useRadioButtons">Use radio buttons</mat-slide-toggle>
     </div>
 
     <div *ngIf="useRadioButtons; else dragAndDrop">
-      <div class="text-header">Comment A</div>
-      <div class="text">{{question.comment_a}}</div>
-      <div class="text-header">Comment B</div>
-      <div class="text">{{question.comment_b}}</div>
-      <div class="question-group">
-        <div class="toxicity-selected-value">
-          <span class="label-heading">How does comment A compare to comment B?</span>
+      <div class="toxicity-selected-value">
+        <span class="label-heading">How do the coments compare?</span>
+      </div>
+      <div class="row">
+        <div>
+          <div class="text-header">Comment A:</div>
+          <div class="text">{{question.comment_a}}</div>
+        </div>
+        <div class="inline-description">is</div>
```

Comment:
w.r.t wrapping, you probably don't want separate bits of the comparative statement wrapping separately. I'd put a <div> around the "is" to the "getComparator", so that you have 3 elements to wrap: comment A, the comparison bit, and comment B. seem right?

Reply:
Done.